### PR TITLE
Quotes

### PR DESCRIPTION
**Pull request by rafaelfragosom**
I've been using this Gulp plugin for a week and my code is using single quotes, so I keep getting this warning from JSHint that the constants file is using double quotes and it doesn't match the code style.

I added another option to the plugin to able developers to set a quote style when generating the file. 

--------------------------------

I wrote 2 more tests and a function to decode HTML string because the `toString()` method was escaping the quotes and making the tests fail. Using this npm plugin [html-entities](https://www.npmjs.com/package/html-entities).

PS: Not sure if I could update the version of the plugin (I didn't).

### CODE REVIEW

**Comment by enkodellc** [comment]
+1

**Comment by guzart** [comment]
Thanks a lot @rafaelfragosom, the commit looks very good. But I think it's missing the possible quotes from strings after the JSON serialization using [`JSON.stringify`](https://github.com/guzart/gulp-ng-constant/blob/master/index.js#L102).

I looked around for a JSON serializer with an option for single/double quotes, but I couldn't find any.  Another solution I can think of would be to use various Regex to replace the quotes, but I am worried that would turn out very complicated.

If it's only because of the JSHint linter, am thinking it might be easier to add a comment that turns off the quote linting.

**Comment by ghachey** [comment]
A way to add the comment that turns off the quote linting would be an acceptable work around for me.

**Comment by rafaelfragosom** [comment]
@guzart I don't understand your answer.

Using the jshint comments will do the job but we all we're doing and that voice inside our heads won't go away lol. I just want to get rid of these annoying warnings. Your call.
